dprint.json

```diff
@@ -8,6 +8,9 @@
     "commands": [{
       "command": "rustfmt --edition 2021",
       "exts": ["rs"]
+    }, {
+      "command": "msgcat -",
```

Comment:
How does this work? Is it piping rustfmt output to msgcat ?

Reply:
Almost, it's piping all `.po` files (from `exts` below) through `msgcat -`. This normalizes the PO files.